Documentation/dt-bindings.md

```diff
@@ -247,3 +247,40 @@ new keycode to the actual key.
 		};
 	};
 ```
+
+### GPIO leds
+
+We also support the `gpio-leds` node. This node follows upstream binding except
```

Comment:
```suggestion
lk2nd supports the `gpio-leds` node. This node follows upstream binding except
```

My personal preference is to write documentation from the point of the software, not the people making it